keith-vscode/src/table/table-webview.ts

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-promise-executor-return */
+/* eslint-disable no-console */
+/* eslint-disable no-return-assign */
+/* eslint-disable prefer-template */
+/* eslint-disable prettier/prettier */
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ *
+ * Copyright 2022 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import * as vscode from 'vscode';
+import { AddRowAction, AddRowListenerAction, ResetTableAction, SelectedRowAction, UpdateCellAction } from './actions';
+
+export class TableWebview {
+
+    protected disposables: vscode.Disposable[] = [];
+
+    protected title: string;
+
+    protected headers: string[];
+    
+    static viewCount = 0;
+
+    readonly identifier: string;
+
+    readonly localResourceRoots: vscode.Uri[];
+
+    readonly scriptUri: vscode.Uri;
+
+    webview: vscode.Webview;
+
+    private resolveWebviewReady: () => void;
+
+    private readonly webviewReady = new Promise<void>((resolve) => this.resolveWebviewReady = resolve);
```

Comment:
here I would also remove the newlines between the attributes

Reply:
Again prettier does not allow this.